samples/tests/playwright.config.ts

```diff
@@ -17,6 +17,8 @@ const config: PlaywrightTestConfig = {
       use: {
         browserName: 'chromium',
         permissions: ['notifications', 'camera', 'microphone'],
+        // Large enough to contain all UI elements from ComponentExamples sample
+        viewport: { width: 900, height: 900 },
```

Comment:
Instead of setting a larger viewport, could we use the playwright API to take a whole page screenshot: https://playwright.dev/docs/screenshots#full-page-screenshots
(from the documentation it appears this should capture elements outside the immediate view, but you would need to test if this works)

Reply:
iirc, we're already taking full page screenshots...

Reply:
.. and this is a change leaked in from a PR that Edward landed this morning. Should disappear if you merge from `origin/main` :)